conf/nomad/minio.hcl

```diff
@@ -24,7 +24,7 @@ job "${service_name}" {
     }
 
   %{ if use_host_volume }
-    volume "persistence" {
+    volume "${host_volume_name}" {
```

Comment:
Do you need an update also mounting of volume https://github.com/fredrikhgrelland/terraform-nomad-minio/blob/master/conf/nomad/minio.hcl#L79 ?

Reply:
See answer in Postgres PR: https://github.com/fredrikhgrelland/terraform-nomad-postgres/pull/58#discussion_r524914677